packages/react-spinner/src/components/Spinner/useSpinnerStyles.ts

```diff
@@ -1,34 +1,254 @@
-import { SlotClassNames } from '@fluentui/react-utilities';
-import { makeStyles, mergeClasses } from '@griffel/react';
-import type { SpinnerSlots, SpinnerState } from './Spinner.types';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { tokens } from '@fluentui/react-theme';
+import type { SpinnerState, SpinnerSlots } from './Spinner.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
 
-/**
- * @deprecated Use `spinnerClassNames.root` instead.
- */
-export const spinnerClassName = 'fui-Spinner';
 export const spinnerClassNames: SlotClassNames<SpinnerSlots> = {
   root: 'fui-Spinner',
+  spinner: 'fui-Spinner__spinner',
+  label: 'fui-Spinner__label',
+};
+
+const rvalues = {
+  tiny: '9',
+  extraSmall: '11',
+  small: '13',
+  medium: '14.5',
+  large: '16.5',
+  extraLarge: '18.5',
+  huge: '20',
+};
+// Sizes for the Spinner
+const spinnnerSizes = {
+  tiny: '20px',
+  extraSmall: '24px',
+  small: '28px',
+  medium: '32px',
+  large: '36px',
+  extraLarge: '40px',
+  huge: '44px',
 };
+
+// Stroke widths for the Spinner
+const spinnerStrokeWidth = {
+  sWidth: '2px',
+  mWidth: '3px',
+  lWidth: '4px',
+};
+
```

Comment:
If these sizes are going to be added to the theme tokens, maybe add a TODO mentioning they should be replaced once the tokens are added.